### DISCUSSION TITLE
Viewing Rest.JS Github Pages on Mobile Browser Does not Display Code Regions

### DISCUSSION THREAD

**Discussion by nisfornature** [octokit/octokit.js#2072] [General] [answered]
When viewing https://octokit.github.io/rest.js/v18 on a mobile browser, preformatted text and code elements are not visible. While developing on mobile browser, mobile users don't have the same visibility as users who are able to accommodate a larger visual viewport dimension.

**Comment by gr2m** [comment, marked as answer]
could you please share a screenshot of what you see? It should render all contents, no matter the screen resolution

<img width="383" alt="image" src="https://user-images.githubusercontent.com/39992/114899787-1ec24400-9dc8-11eb-8905-4d88663e9f25.png">

**Reply by nisfornature** [reply]
My mistake. Sorry for not having provided a screenshot initally. I can't reproduce the issue. I can see code elements of varying line width adjust under the paragraph, and the issue I believe experienced were a few code segments having text out-of-view and displayed interwoven with paragraph comments.

**Reply by gr2m** [reply]
no worries, thank you for confirming that it's working.
